pkg/apis/application/v1alpha1/types.go

```diff
@@ -439,6 +443,133 @@ type SyncPolicyAutomated struct {
 	SelfHeal bool `json:"selfHeal,omitempty" protobuf:"bytes,2,opt,name=selfHeal"`
 }
 
+func (s *SyncPolicy) IsAutomated() bool {
+	return s != nil && s.Automated != nil
+}
+
+// Maintenance controls when syncs can be run
+type Maintenance struct {
+	// Enabled will allow maintenance windows to run during their scheduled times (default: true)
+	Enabled bool `json:"enabled,enabled" protobuf:"bytes,1,opt,name=enabled"`
```

Comment:
good stuff, I think there maybe a typo on this line? 

```
Enabled bool `json:"enabled,omitempty" protobuf:"bytes,1,opt,name=enabled"`
```

Reply:
Thanks. Will update.